src/Components/DogList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { fetchDogs } from '../services/dogs';
+import { Link } from 'react-router-dom';
+export default function DogList() {
+  const [dogs, setDogs] = useState([]);
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchDogs();
+      setDogs(data);
+      setLoading(false);
```

Comment:
It appears this function is not returning any dogs. Could it be a problem with your secrets on netlify or github?